js/views/modals/listingDetail/Listing.js

```diff
@@ -168,13 +282,30 @@ export default class extends BaseModal {
         shipsFreeToMe: this.shipsFreeToMe,
         ownListing: this.model.isOwnListing,
         displayCurrency: app.settings.get('localCurrency'),
+        shipsFromCountry: app.settings.get('country'),
       }));
 
       super.render();
 
       this._$deleteListing = null;
       this._$shipsFreeBanner = null;
       this._$popInMessages = null;
+      this._$photoSection = null;
+
+      // commented out until variants are available
+      // this.$('.js-variantSelect').select2();
+
+      this.$photoSelected = this.$('.js-photoSelected');
+      this.$photoSelected.backgroundDraggable();
+
+      const shippingDest = this.$('#shippingDestinations');
+
+      shippingDest.select2({
+        data: this.select2CountryData,
+        placeholder: app.polyglot.t('listingDetail.shipToPlaceholder'),
+      });
+
+      shippingDest.val(app.settings.get('country')).trigger('change');
```

Comment:
I think select2 is doing a poor practice. Why would a module, instead of calling one of its functions directly, take the circuitous route of programmatically firing an event handler and having the code path go from point A to B to C to D... (remember, it's going to go through jQuery event handling code) instead of point A to B.

I won't hold up the merge if you don't think it's important, I'm just flagging what IMO is not clean code.